Pythonwin/Scintilla/src/LexGen.py

```diff
@@ -135,8 +135,8 @@ def Generate(inpath, outpath, commentPrefix, eolType, *lists):
     # print "generate '%s' -> '%s' (comment prefix: %r, eols: %r)"\
     #      % (inpath, outpath, commentPrefix, eolType)
     try:
-        infile = open(inpath, "r")
```

Comment:
I know Scintilla is vendored, but these changes are automated. Just like `pycln`, `isort` and `black`, is that fine? I assume the idea is that we trust this won't lead to behaviour changes and that updating Scintilla won't cause these changes to be lost because they'll be automatically re-applied.

Reply:
The problem then will be that re-vendoring in the future would break, meaning we'd need to work out what to re-run etc. It seems better long term to just ignore scintilla entirely, which works both today *and* in the future.